@vktng 

I tried manually --> and I still see some installation errors:

```
[root@47d91f52c7a6 magma]# bazel build //lte/gateway/c/core:mme_oai --define=disable_sentry_native=1 
2022/11/29 15:21:05 Downloading https://releases.bazel.build/5.2.0/release/bazel-5.2.0-linux-x86_64...
Extracting Bazel installation...
Starting local Bazel server and connecting to it...
INFO: Invocation ID: e97d00bb-377d-4101-9adf-35460e318659
INFO: Reading 'startup' options from /magma/.bazelrc: --output_base=/var/tmp/bazel, --host_jvm_args=-Xmx8g
INFO: Options provided by the client:
  Inherited 'common' options: --isatty=1 --terminal_columns=203
INFO: Reading rc options for 'build' from /magma/.bazelrc:
  Inherited 'common' options: --experimental_allow_tags_propagation
INFO: Reading rc options for 'build' from /magma/bazel/bazelrcs/cache.bazelrc:
  Inherited 'common' options: --config=disk_cache
INFO: Reading rc options for 'build' from /magma/.bazelrc:
  'build' options: --announce_rc --color=yes --cxxopt=-std=c++14 --strip=never --per_file_copt=^lte/gateway/c/.*$@-g --test_env=PATH=/bin:/usr/bin:/usr/local/bin:/usr/sbin --test_env=MAGMA_ROOT --test_env=S1AP_TESTER_ROOT --test_env=GO_TEST_WRAP_TESTV=1 --per_file_copt=^lte/gateway/c/core/.*$@-DPACKAGE_BUGREPORT="TBD" --per_file_copt=^lte/gateway/c/core/.*$@-DPACKAGE_VERSION="0.1" --per_file_copt=^lte/gateway/c/core/oai/tasks/amf/.*$@-DPACKAGE_NAME="AMF" --per_file_copt=^lte/gateway/c/core/oai/tasks/mme_app/.*$@-DPACKAGE_NAME="MME" --per_file_copt=^lte/gateway/c/core/oai/tasks/sgw/.*$@-DPACKAGE_NAME="S/P-GW" --javacopt=-source 8 --javacopt=-target 8 --java_runtime_version=remotejdk_11
INFO: Found applicable config definition common:disk_cache in file /magma/.bazelrc: --repository_cache=/var/cache/bazel-cache-repo
INFO: Found applicable config definition build:disk_cache in file /magma/.bazelrc: --disk_cache=/var/cache/bazel-cache
ERROR: Traceback (most recent call last):
	File "/var/tmp/bazel/external/system_libraries/BUILD.bazel", line 78, column 16, in <toplevel>
		srcs = glob(
Error in glob: glob pattern 'usr/lib/*-linux-gnu/libglog.so.0' didn't match anything, but allow_empty is set to False (the default value of allow_empty can be set with --incompatible_disallow_empty_glob).
ERROR: /magma/lte/gateway/c/core/BUILD.bazel:1170:11: Target '@system_libraries//:czmq' contains an error and its package is in error and referenced by '//lte/gateway/c/core:lib_mme_oai'
ERROR: /magma/lte/gateway/c/core/BUILD.bazel:1170:11: Target '@system_libraries//:libconfig' contains an error and its package is in error and referenced by '//lte/gateway/c/core:lib_mme_oai'
ERROR: /magma/lte/gateway/c/core/BUILD.bazel:1170:11: Target '@system_libraries//:libfd' contains an error and its package is in error and referenced by '//lte/gateway/c/core:lib_mme_oai'
ERROR: /magma/lte/gateway/c/core/BUILD.bazel:1170:11: no such target '@system_libraries//:libgnutls': target 'libgnutls' not declared in package '' (did you mean 'libmnl'?) defined by /var/tmp/bazel/external/system_libraries/BUILD.bazel and referenced by '//lte/gateway/c/core:lib_mme_oai'
ERROR: /magma/lte/gateway/c/core/BUILD.bazel:1170:11: Target '@system_libraries//:libnettle' contains an error and its package is in error and referenced by '//lte/gateway/c/core:lib_mme_oai'
ERROR: /magma/lte/gateway/c/core/BUILD.bazel:1170:11: no such target '@system_libraries//:libsqlite3-dev': target 'libsqlite3-dev' not declared in package '' defined by /var/tmp/bazel/external/system_libraries/BUILD.bazel and referenced by '//lte/gateway/c/core:lib_mme_oai'
ERROR: /magma/lte/gateway/c/core/BUILD.bazel:1170:11: no such target '@system_libraries//:libsystemd': target 'libsystemd' not declared in package '' defined by /var/tmp/bazel/external/system_libraries/BUILD.bazel and referenced by '//lte/gateway/c/core:lib_mme_oai'
ERROR: Analysis of target '//lte/gateway/c/core:mme_oai' failed; build aborted: 
INFO: Elapsed time: 20.340s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (56 packages loaded, 239 targets configured)
```

--- a/ci-scripts/docker/Dockerfile.mme.ci.rhel8
+++ b/ci-scripts/docker/Dockerfile.mme.ci.rhel8
@@ -18,15 +18,15 @@ COPY ./ $MAGMA_ROOT
 RUN export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:/usr/local/lib/pkgconfig/ && \
     # Remove entitlements
     rm -Rf $MAGMA_ROOT/etc-pki-entitlement $MAGMA_ROOT/rhsm-conf $MAGMA_ROOT/rhsm-ca && \
-    cd $MAGMA_ROOT/lte/gateway && \
+    cd $MAGMA_ROOT && \
     echo $FEATURES && \
-    make build_oai && \
-    make build_sctpd && \
-    cp $C_BUILD/core/oai/oai_mme/mme $C_BUILD/core/oai/oai_mme/oai_mme && \
+    bazel build //lte/gateway/c/core:mme_oai --define=disable_sentry_native=1 && \
+    bazel build //lte/gateway/c/sctpd/src:sctpd --define=disable_sentry_native=1 && \
+    mv $MAGMA_ROOT/bazel-bin/lte/gateway/c/core/mme_oai $MAGMA_ROOT/bazel-bin/lte/gateway/c/core/oai_mme && \
     echo 'Shared libraries for oai_mme' && \
-    ldd $C_BUILD/core/oai/oai_mme/oai_mme && \
+    ldd $MAGMA_ROOT/bazel-bin/lte/gateway/c/core/oai_mme && \
     echo 'Shared libraries for sctpd' && \
-    ldd $C_BUILD/sctpd/src/sctpd
+    ldd $MAGMA_ROOT/bazel-bin/lte/gateway/c/sctpd/src/sctpd
 
 # Prepare config file
 RUN cd $MAGMA_ROOT/lte/gateway/docker/mme/configs/ && \
@@ -115,9 +115,9 @@ COPY --from=magma-mme-builder \
     /usr/local/lib/libfdcore.so.6 \
     /usr/local/lib/libfdproto.so.6 \
 # From nettle/gnutls src build
-    /usr/local/lib/libgnutls.so.28 \
-    /usr/local/lib/libnettle.so.4 \
-    /usr/local/lib/libhogweed.so.2 \
+    /lib64/libgnutls.so.28 \
+    /lib/libnettle.so.4 \
+    /lib/libhogweed.so.2 \
     /lib64/
 
 # Copy all fdx files from freeDiameter installation
@@ -130,8 +130,8 @@ RUN ldconfig
 # Copy pre-built binaries for MME and SCTPD
 WORKDIR /magma-mme/bin
 COPY --from=magma-mme-builder \
-    $C_BUILD/core/oai/oai_mme/oai_mme \
-    $C_BUILD/sctpd/src/sctpd \
+    $MAGMA_ROOT/bazel-bin/lte/gateway/c/core/oai_mme \
+    $MAGMA_ROOT/bazel-bin/lte/gateway/c/sctpd/src/sctpd \
     ./
 
 # Copy the configuration file templates and mean to modify/generate certificates

--- a/lte/gateway/docker/mme/Dockerfile.rhel8
+++ b/lte/gateway/docker/mme/Dockerfile.rhel8
@@ -225,7 +225,7 @@ RUN tar -xf nettle-2.5.tar.gz && \
     cd nettle-2.5 && \
     mkdir _build && \
     cd _build/ && \
-    ../configure --libdir=/usr/local/lib && \
+    ../configure --libdir=/usr/lib && \
     make -j`nproc` && \
     make install && \
     ldconfig -v && \
@@ -234,7 +234,7 @@ RUN tar -xf nettle-2.5.tar.gz && \
     # Moved wget https://www.gnupg.org/ftp/gcrypt/gnutls/v3.1/gnutls-3.1.23.tar.xz && \
     tar xf gnutls-3.1.23.tar.xz && \
     cd gnutls-3.1.23 && \
-    ./configure --with-libnettle-prefix=/usr/local && \
+    ./configure --with-libnettle-prefix=/usr --prefix=/usr && \
     sed -i -e "s#elif 0#elif 1#" gl/fseterr.c && \
     make -j`nproc` && \
     make install && \
@@ -302,6 +302,17 @@ RUN cd json && \
     cd ../ && rm -Rf _build && \
     ldconfig --verbose
 
+# Install bazel
+WORKDIR /usr/sbin
+RUN wget --progress=dot:giga https://github.com/bazelbuild/bazelisk/releases/download/v1.10.0/bazelisk-linux-amd64 && \
+    chmod +x bazelisk-linux-amd64 && \
+    ln -sf /usr/sbin/bazelisk-linux-amd64 /usr/sbin/bazel
+
+RUN mkdir -p /usr/lib/x86_64-linux-gnu/ && \
+    ln -s /lib64/libglog.so /usr/lib/x86_64-linux-gnu/libglog.so.0 && \
+    ln -s /lib64/libgnutls.so /usr/lib/libgnutls.so && \
+    ldconfig --verbose
+
 # Starting from here, we are building MAGMA services (sctpd and mme)
 FROM magma-mme-base as magma-mme-builder
 
@@ -312,15 +323,15 @@ COPY ./ $MAGMA_ROOT
 RUN export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:/usr/local/lib/pkgconfig/ && \
     # Remove entitlements
     rm -Rf $MAGMA_ROOT/etc-pki-entitlement $MAGMA_ROOT/rhsm-conf $MAGMA_ROOT/rhsm-ca && \
-    cd $MAGMA_ROOT/lte/gateway && \
+    cd $MAGMA_ROOT && \
     echo $FEATURES && \
-    make build_oai && \
-    make build_sctpd && \
-    cp $C_BUILD/core/oai/oai_mme/mme $C_BUILD/core/oai/oai_mme/oai_mme && \
+    bazel build //lte/gateway/c/core:mme_oai --define=disable_sentry_native=1 && \
+    bazel build //lte/gateway/c/sctpd/src:sctpd --define=disable_sentry_native=1 && \
+    mv $MAGMA_ROOT/bazel-bin/lte/gateway/c/core/mme_oai $MAGMA_ROOT/bazel-bin/lte/gateway/c/core/oai_mme && \
     echo 'Shared libraries for oai_mme' && \
-    ldd $C_BUILD/core/oai/oai_mme/oai_mme && \
+    ldd $MAGMA_ROOT/bazel-bin/lte/gateway/c/core/oai_mme && \
     echo 'Shared libraries for sctpd' && \
-    ldd $C_BUILD/sctpd/src/sctpd
+    ldd $MAGMA_ROOT/bazel-bin/lte/gateway/c/sctpd/src/sctpd
 
 # Prepare config file
 RUN cd $MAGMA_ROOT/lte/gateway/docker/mme/configs/ && \
@@ -354,8 +365,8 @@ FROM magma-mme-builder as magma-dev-mme
 
 # Copy the configuration file templates and mean to modify/generate certificates
 WORKDIR /magma-mme/bin
-RUN cp $C_BUILD/core/oai/oai_mme/mme oai_mme
-RUN cp $C_BUILD/sctpd/src/sctpd .
+RUN cp $MAGMA_ROOT/bazel-bin/lte/gateway/c/core/oai_mme .
+RUN cp $MAGMA_ROOT/bazel-bin/lte/gateway/c/sctpd/src/sctpd .
 WORKDIR /magma-mme/etc
 RUN cp $MAGMA_ROOT/lte/gateway/docker/mme/configs/mme.conf .
 RUN cp $MAGMA_ROOT/lte/gateway/docker/mme/configs/mme_fd.conf .
@@ -441,9 +452,9 @@ COPY --from=magma-mme-builder \
     /usr/local/lib/libfdcore.so.6 \
     /usr/local/lib/libfdproto.so.6 \
 # From nettle/gnutls src build
-    /usr/local/lib/libgnutls.so.28 \
-    /usr/local/lib/libnettle.so.4 \
-    /usr/local/lib/libhogweed.so.2 \
+    /lib64/libgnutls.so.28 \
+    /lib/libnettle.so.4 \
+    /lib/libhogweed.so.2 \
     /lib64/
 
 # Copy all fdx files from freeDiameter installation
@@ -456,8 +467,8 @@ RUN ldconfig
 # Copy pre-built binaries for MME and SCTPD
 WORKDIR /magma-mme/bin
 COPY --from=magma-mme-builder \
-    $C_BUILD/core/oai/oai_mme/oai_mme \
-    $C_BUILD/sctpd/src/sctpd \
+    $MAGMA_ROOT/bazel-bin/lte/gateway/c/core/oai_mme \
+    $MAGMA_ROOT/bazel-bin/lte/gateway/c/sctpd/src/sctpd \
     ./
 
 # Copy the configuration file templates and mean to modify/generate certificates

